.github/workflows/rubocop.yml

```diff
@@ -9,10 +9,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.6
-    - run: gem install rubocop rubocop-rubycw --no-document
```

Comment:
[note] `--no-document` is unnecessary:
https://github.com/soutaro/querly/pull/86/checks#step:4:14